test/e2e/inspect_test.go

```diff
@@ -117,6 +117,18 @@ var _ = Describe("Podman inspect", func() {
 		Expect(len(result.OutputToStringArray())).To(Equal(1))
 	})
 
+	It("podman inspect container and filter for OSArguments", func() {
+		SkipIfRemote()
+		ls, ec, _ := podmanTest.RunLsContainer("")
+		Expect(ec).To(Equal(0))
+		cid := ls.OutputToString()
+
+		result := podmanTest.Podman([]string{"inspect", "--format={{.Config.OSArguments}}", cid})
+		result.WaitWithDefaultTimeout()
+		Expect(result.ExitCode()).To(Equal(0))
+		Expect(len(result.OutputToStringArray())).To(Equal(1))
+	})
+
```

Comment:
Would be nice to add at some point a check to make sure the contained value is what you expect it to be.